### PR TITLE
small analysis of result table

### PR DESCRIPTION
**Pull request by rasmus-e-christensen**

### CODE REVIEW

**Review by MrLongbottom — APPROVED** [review]

File: tables/metric_results.tex [hunk 1]
```diff
@@ -10,4 +10,10 @@
 		Taxonomy-Topic Model & - & 0.660 & 0.709 \\
 	\end{tabular}
 	\label{tab:metric_results}
-\end{table}
\ No newline at end of file
+\end{table}
+
+From \autoref{tab:metric_results}, we can see that the Author and Category-Topic models are performing the worst, whereas the Taxonomy-Topic model is out performing all other models.
```
Comment (Starrock45) [inline tables/metric_results.tex]:
```suggestion
From \autoref{tab:metric_results}, we can see that the Author and Category-Topic models are performing the worst, whereas the Taxonomy-Topic model is outperforming all other models.
```

File: tables/metric_results.tex [hunk 2]
```diff
@@ -10,4 +10,10 @@
...
+However, the run time of the Taxonomy model is worse than the standard \gls{lda}.
+For the \gls{lda} to compute 50 epochs, that takes around 6-8 hours, depending on the CPU.
```
Comment (Starrock45) [inline tables/metric_results.tex]:
```suggestion
It takes about 6-8 hours to compute 50 epochs for the \gls{lda} model, depending on the CPU. 
```

File: tables/metric_results.tex [hunk 3]
```diff
@@ -10,4 +10,10 @@
...
 	\end{tabular}
 	\label{tab:metric_results}
-\end{table}
\ No newline at end of file
+\end{table}
+
+From \autoref{tab:metric_results}, we can see that the Author and Category-Topic models are performing the worst, whereas the Taxonomy-Topic model is out performing all other models.
+However, the run time of the Taxonomy model is worse than the standard \gls{lda}.
+For the \gls{lda} to compute 50 epochs, that takes around 6-8 hours, depending on the CPU. 
+The Taxonomy model running a 5 layered pachinko model, took 132 hours before completing the 50 epochs.
```
Comment (Starrock45) [inline tables/metric_results.tex]:
```suggestion
The Taxonomy model running a 5 layered \gls{pam} took 132 hours before completing the 50 epochs.
```

**Review by Starrock45 — APPROVED** [review]
Enkelte ting

**Pull request merged**